hack/podmansnoop

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# @lint-avoid-python-3-compatibility-imports
+from __future__ import print_function
+
+# Based on the `exitsnoop` scripts at https://github.com/iovisor/bcc/blob/master/tools/exitsnoop.py
+
+import argparse
+import os
+import platform
+import re
+import signal
+import sys
+
+from bcc import BPF
+from datetime import datetime
+from time import strftime
+
+def _embedded_c():
+    """Generate C program for sched_process_exit tracepoint in kernel/exit.c."""
+    c = """
+#include <uapi/linux/ptrace.h>
+#include <linux/sched.h>
+
+#define ARGSIZE  128
+
+struct data_t {
+    u64 start_time;
+    u64 exit_time;
+    u32 pid;
+    u32 tid;
+    u32 ppid;
+    u32 sig_info;
+    char task[TASK_COMM_LEN];
+    char argv[ARGSIZE];
+};
+
+BPF_PERF_OUTPUT(events);
+
+TRACEPOINT_PROBE(sched, sched_process_exit)
+{
+    struct task_struct *task = (typeof(task))bpf_get_current_task();
+    if (FILTER_PID) { return 0; }
+
+    struct data_t data = {};
+
+    data.start_time = PROCESS_START_TIME_NS,
+    data.exit_time = bpf_ktime_get_ns(),
+    data.pid = task->tgid,
+    data.tid = task->pid,
+    data.ppid = task->real_parent->tgid,
+    data.sig_info = task->exit_code & 0xFF,
+    bpf_get_current_comm(&data.task, sizeof(data.task));
+
+    events.perf_submit(args, &data, sizeof(data));
+    return 0;
+}
+"""
+
+    filter_pid = "!(task->pid == task->tgid)"
+
+    code_substitutions = [
+        ('FILTER_PID', filter_pid),
+        ('PROCESS_START_TIME_NS', 'task->start_time'),
+    ]
+    for old,new in code_substitutions:
+        c = c.replace(old, new)
+    return c
+
+def _print_header():
+    print("%-16s %-7s %-7s %-7s %-7s" %
+              ("PCOMM", "PID", "PPID", "TID", "AGE(ms)"))
+
+buffer = None
+
+def _print_event(cpu, data, size): # callback
+    """Print the exit event."""
+    global buffer
+    e = buffer["events"].event(data)
+
+    task = e.task.decode()
+    if task == "3":
+        # For absolutely unknown reasons, 'crun' appears as '3'.
+        task = "crun"
+
+    if task not in ["podman", "crun", "runc", "conmon"]:
```

Comment:
Can you add netavark and aardvark-dns as well? Might be useful as well I made good improvements in netavark recently and there is more we can do over there as well.